test/Table_Tests/src/Common_Table_Operations/Select_Columns_Spec.enso

```diff
@@ -250,23 +261,23 @@ spec setup =
             err.should_fail_with Column_Indexes_Out_Of_Range
 
         Test.specify "should correctly handle edge-cases: duplicate indices" <|
-            selector = By_Index [0, 0, 0]
+            selector = [0, 0, 0]
             t = table.reorder_columns selector Position.After_Other_Columns on_problems=Problem_Behavior.Report_Error
             expect_column_names ["bar", "Baz", "foo_1", "foo_2", "ab.+123", "abcd123", "foo"] t
 
         Test.specify "should correctly handle edge-cases: aliased indices" <|
-            selector = By_Index [0, -7, -6, 1]
+            selector = [0, -7, -6, 1]
             t = table.reorder_columns selector Position.After_Other_Columns on_problems=Problem_Behavior.Report_Error
             expect_column_names ["Baz", "foo_1", "foo_2", "ab.+123", "abcd123", "foo", "bar"] t
 
         Test.specify "should correctly handle edge-cases: duplicate names" <|
```

Comment:
I'd add a test case where different kinds of selectors are mixed. Now this is permitted, so we should verify it works (looking at the code it looks like it should, but adding a test we will ensure we don't regress)

Reply:
I have

Reply:
Oh, sorry! I must have scrolled past it.